Autocreate

bmccallister opened the pull request:
This wraps all methods in autocreation prechecks.  If the autoCreate flag is set on that version of the schema, the call will call list tables and determine if the table should be created prior to execution.

stephen-ta commented:
:accept: Great job!

stephen-ta commented:
Not sure if you are still doing version numbers, but might want to start putting that back again in the title and then a description for the changes.

Fluidbyte commented:
Only thing I see - add a note about usage in the `README` so it's visible to other devs without needing to dive the specs.

bmccallister commented:
I've added the info to the readme, and I also changed the lerr to err in the helpers file, because the name change was no longer necessary and I'd like it to obey convention and not cause questions.

Fluidbyte commented on README.md:
````diff
@@ -29,12 +29,13 @@ let dynamoConfig = {
 ```
 
-Add an instance of the model
+Add an instance of the model where the autoCreate flag is used to determine if calls to non-existant tables should automatically create the table from the model
````
...where the `autoCreate` flag...

Fluidbyte commented:
:accept:
